s4e-web/package.json

```diff
@@ -8,7 +8,7 @@
     "start-notifications": "ng build notifications --watch",
     "build-libraries": "ng build notifications",
     "prestart": "npm run make-requirements",
-    "start": "ng serve --proxy-config proxy.conf.json --configuration hmr",
+    "start": "ng serve --host 192.168.1.10 --proxy-config proxy.conf.json --configuration hmr",
```

Comment:
Why are you running ng start at this specific host? Are you sure that other developers will correctly run this command?

Reply:
I'm almost certain that this should not be here...